### ISSUE TITLE
When live-reload=false, don't complain for missing allow-navigation href="*"

### ISSUE THREAD

**Issue by ffissore** [isleofcode/ember-cordova#162]
As of 0.3.3, even when running with `ember cdv:serve --live-reload=false --platform=browser --port 4203 --env development`, it fails complaining with
```
Your ember-cordova/cordova/config.xml file needs the following flag:
	<allow-navigation href="*" />
This is unsafe and should not be used in production,
but is necessary for device live-reload to work.
```

If live reload is disabled, no check should be done on config.xml

**Comment by alexblom** [comment]
This is definitely an issue. Thanks for reporting.

**Comment by alexblom** [comment]
Fixed in 0.3.4 :)
